packages/salesforcedx-vscode-apex/snippets/apex.json

```diff
@@ -181,5 +181,119 @@
             "throw new ${1:Exception}($0)"
         ],
         "description": "Throw a new exception"
+    },
+    "list": {
+        "prefix": "list<",
+        "body": [
+            "list<${1:object}> ${2:listName} = new list<$1>();"
+        ],
+        "description": "list constructor"
+    },
+    "map": {
+        "prefix": "map<",
+        "body": [
+            "map<${1:object}, ${2:object}> ${3:mapName} = new map<$1, $2>();"
+        ],
+        "description": "map constructor"
+    },
+    "set": {
+        "prefix": "set<",
+        "body": [
+            "set<${1:object}> ${2:setName} = new set<$1>();"
+        ],
+        "description": "set constructor"
+    },
+    "@AuraEnabled": {
+        "prefix": "@AuraEnabled",
+        "body": [
+            "@AuraEnabled",
+            "${1|public,global|} static ${2:string} ${3:methodName}(){",
+            "\t$4",
+            "}"
+        ],
+        "description": "Aura Enabled Apex Method"
+    },
+    "userCreation": {
+        "prefix": "createUsers",
+        "body": [
+            "public static list<user> createUsers(integer numberOfUsers){",
+            "\tlist<user> users = new list<user>();",
```

Comment:
Rather than using the \t the preferred style is to actually include spaces in each line (4 spaces for each level of indent). This makes the snippet more readable. So this would be:

```
"public static list<user> createUsers(integer numberOfUsers){",
"    list<user> users = new list<user>();",
"    Profile p = [SELECT Id FROM Profile WHERE Name='${1:Standard User}'];",
"    for (integer i=0; i<numberOfUsers; i++){",
"        string x = string.valueOf(i);",
"        User u = new User(Alias = 't'+x, Email='standarduser'+x+'@${2:someDomain.test}',",
```

Reply:
Use `List` with a capital letter.